internal/kibana/status.go

```diff
@@ -56,3 +61,26 @@ func (c *Client) requestVersion() (VersionInfo, error) {
 
 	return status.Version, nil
 }
+
+// CheckHealth checks the Kibana health
+func (c *Client) CheckHealth() error {
```

Comment:
Maybe we can use the same internal method to get the version and the health.

Reply:
I renamed that internal method to be `requestStatus` and change the required usages.